### PR TITLE
Translate .../getting-started/anatomy-of-an-ember-app.md

### LINKED ISSUE #48: Translate `/getting-started/anatomy-of-an-ember-app.md`, Ember 4.7
Please assign yourself to the issue or let a comment at the very moment you start the translation.<br><br>File: `guides/release/getting-started/anatomy-of-an-ember-app.md`<br>From Ember: **N/A**<br>To: **4.7**

### PR DESCRIPTION
**Pull request by BlueCutOfficial**
Closes #48

### CODE REVIEW

**Review by Agathebadia — COMMENTED** [review]
line 12 - cliquant - instead of clicquant
line 81 - alt='image de profil'

File: guides/release/getting-started/anatomy-of-an-ember-app.md [hunk 1]
```diff
@@ -1,130 +1,104 @@
...
 
-* The user loads the app for the first time.
-* The user changes the URL manually, such as by clicking the back button or by editing the address bar.
-* The user clicks a link within the app.
-* Some other event in the app causes the URL to change.
+Il existe plusieurs manières de modifier l'URL courante :
 
-No matter how the URL gets set, the first thing that happens is that the Ember router maps the URL to a route handler.
+- L'utilisateur charge l'application pour la première fois.
+- L'utilisateur change l'URL manuellement, en clicquant sur le bouton "Précédent" ou en éditant la barre d'adresse du navigateur.
```
Comment (BlueCutOfficial) [inline guides/release/getting-started/anatomy-of-an-ember-app.md:12]:
```suggestion
- L'utilisateur change l'URL manuellement, en cliquant sur le bouton "Précédent" ou en éditant la barre d'adresse du navigateur.
```

File: guides/release/getting-started/anatomy-of-an-ember-app.md [hunk 2]
```diff
@@ -1,130 +1,104 @@
...
+Dans sa forme la plus basique, un composant est juste un morceau de template identifié par son nom. On peut assimiler les composants aux fonctions des langages de programmation, car ils peuvent aussi prendre des paramètres d'entrée (_arguments_), ce qui les rend adaptables au contexte spécifique dans lequel ils sont affichés.
 
-For example, the example in the previous section is getting a bit long. We can
-_extract_ the snippet for rendering the user's name and profile picture into
-its own component:
+Reprenons l'exemple de la partie précédente, qui commence à contenir un peu de complexité. Nous pouvons "extraire" le code qui affiche le nom de l'utilisateur et sa photo de profil dans un composant dédié :
 
 ```handlebars {data-filename="app/components/user-profile.hbs"}
-<img src="{{@user.profileImage}}" alt="{{@user.name}}'s profile picture"> {{@user.name}}
+<img src="{{@user.profileImage}}" alt="image de profile de {{@user.name}}" />
```
Comment (BlueCutOfficial) [inline guides/release/getting-started/anatomy-of-an-ember-app.md:81]:
```suggestion
<img src="{{@user.profileImage}}" alt="image de profil de {{@user.name}}" />
```

**Comment by BlueCutOfficial** [comment]
@Agathebadia Thanks a lot for the review and for spotting these typos 🙏  

A tiny tip that could save you some time: I recommend to left your comments directly on the concerned line with a suggestion syntax. 
- You can target one or several lines with the plus button
- Then the "+/- file" icon represents a diff, clicking on it will generate for you a suggestion block that you can edit directly
- If (for some reason) I don't want to consider 100% of your suggestion but only a part of it, I can perfectly answer your suggestion with a new one.

Capture d’écran 2023-03-30 à 17 56 58

🙏 Sorry if you already know all of this and you do not use suggestions on purpose. I know your company works on Bitbucket, that's why I thought you may not know that feature.

**Comment by Agathebadia** [comment]
Yes sorry for that, Github is weird atm with me, I could not add a proper review as usual, and added a comment here. Not sure if it's related to my token renewal :S

**Review by Agathebadia — APPROVED** [review]
LGTM!

**Pull request merged**